Lib/test/test_zipimport.py

```diff
@@ -755,6 +791,7 @@ def _testBogusZipFile(self):
 
             error = zipimport.ZipImportError
             self.assertEqual(z.find_module('abc'), None)
+            self.assertEqual(z.find_spec('abc'), None)
```

Comment:
Super minor: the original check here likely predates `assertIs`, so maybe switch both of them over?